### PR TITLE
Add menus using Events (task #1952)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

**Comment by leonidsl** [comment]
:+1:

File: src/Template/Element/View/index.ctp [hunk 1]
```diff
@@ -63,9 +69,16 @@ if (empty($options['title'])) {
                             </td>
                         <?php endforeach; ?>
                         <td class="actions">
-                            <?= $this->Html->link('', ['action' => 'view', $entity->id], ['title' => __('View'), 'class' => 'btn btn-default glyphicon glyphicon-eye-open']) ?>
-                            <?= $this->Html->link('', ['action' => 'edit', $entity->id], ['title' => __('Edit'), 'class' => 'btn btn-default glyphicon glyphicon-pencil']) ?>
-                            <?= $this->Form->postLink('', ['action' => 'delete', $entity->id], ['confirm' => __('Are you sure you want to delete # {0}?', $entity->id), 'title' => __('Delete'), 'class' => 'btn btn-default glyphicon glyphicon-trash']) ?>
+                            <?php
+                                $event = new Event('View.Index.Menu.Actions', $this, [
```
Comment (flangofas) [inline src/Template/Element/View/index.ctp]: @georgeconstantinou An event wrapper element might be handy here to keep DRY principle.

**Comment by flangofas** [comment]
:+1:

File: src/Template/Element/View/index.ctp (hunk 1 above)
Comment (mamchenkov) [inline src/Template/Element/View/index.ctp]: @flangofas agreed.  These are some of the quick fixes that we need for the meeting/demo later today, that's why I merged them.  In general, we'll need to find a better way to call events, with less hard-coding of event names and some generic/consistent way for the parameters.

Good catch! ;)

**Pull request merged**